docker/run_isaac_docker.sh

```diff
@@ -13,7 +13,6 @@ docker run -it --rm --privileged \
 	-e "PRIVACY_CONSENT=Y" \
 	-e "TERM=xterm-256color" \
 	-v "/tmp/.X11-unix:/tmp/.X11-unix:rw" \
-	-v ~/docker/isaac-sim/cache/kit:/isaac-sim/kit/cache:rw \
```

Comment:
Can we leave a comment somewhere else here in the script stating why we are not binding this?
We will probably forget about it.

Reply:
Done!